proposals/csharp-9.0/native-integers.md

```diff
@@ -83,7 +88,7 @@ The tables below cover the conversions between special types.
 | Operand | Target | Conversion | IL |
 |:---:|:---:|:---:|:---:|
 | `nint` | `object` | Boxing | `box` |
-| `nint` | `void*` | PointerToVoid | `conv.i` |
+| `nint` | `void*` | PointerToVoid | `conv.i`/`conv.ovf.u` |
```

Comment:
Consider adding spaces around `/` for consistency.

Reply:
Same as for the other two, in an `unchecked` context I think this could just be a `nop`.

Reply:
And same for `nint to nuint` right below on L92